manuscript/1-SoftReq.txt

```diff
@@ -235,14 +235,14 @@
 Συνήθως υπάρχουν πολλοί άνθρωποι που εμπλέκονται, εκτός από τον ειδικό
 απαιτήσεων, καθένας από τους οποίους έχει μερίδιο στο λογισμικό.
 Τα ενδιαφερόμενα μέρη ποικίλλουν μεταξύ των έργων, περιλαμβάνουν όμως
-πάντα τους χρήστες / διαχειριστές και τους πελάτες (που δεν χρειάζεται
+πάντα τους χρήστες/ διαχειριστές και τους πελάτες (που δεν χρειάζεται
 να είναι οι ίδιοι).
 Τυπικά παραδείγματα των ενδιαφερομένων μερών λογισμικού περιλαμβάνουν
 (αλλά δεν περιορίζονται σε αυτά) τα εξής:
 
   * Χρήστες: Αυτή η ομάδα αποτελείται από αυτούς που θα λειτουργήσουν
   το λογισμικό.
-  Ως επί το πλείστον είναι μια ετερογενής ομάδα που αφορά ανθρώπους
+  Ως επί το πλείστων είναι μια ετερογενής ομάδα που αφορά ανθρώπους
```

Comment:
Please keep the original one.
https://el.wiktionary.org/wiki/%CF%89%CF%82_%CE%B5%CF%80%CE%AF_%CF%84%CE%BF_%CF%80%CE%BB%CE%B5%CE%AF%CF%83%CF%84%CE%BF%CE%BD